worf/validators.py

```diff
@@ -118,6 +118,19 @@ def _validate_positive_int(self, key):
             )
 
         return integer
+    
+    def _validate_decimal(self, key):
+        value = self.bundle[key]
+        
+        if value is None or value == "":
+            return None
+
+        try:
+            decimal = float(value)
```

Comment:
Since the Django DecimalField is backed by an actual Python Decimal class object, I feel like this should be trying to coerce the value into an actual Decimal object for consistency.

Refs:

* https://docs.djangoproject.com/en/4.2/ref/models/fields/#decimalfield
* https://docs.python.org/3/library/decimal.html#decimal.Decimal

Reply:
agreed